DMTR-231.tex

```diff
@@ -53,36 +53,25 @@ \section{Introduction}
 \subsection{Objectives}
 \label{sect:objectives}
 
- This test plan verifies that DM software is ready to obtain and process
-ComCam observations. ~Since this test campaign is needed prior to on-sky
-data acquisition the tests are necessarily focused on the ability to
-process test-stand data. ~Therefore, the elements focus on the generic
-ability to process ComCam data.
-
+ This test plan verifies that DM software and services are ready for on-sky commissioning. 
 
 
 \subsection{System Overview}
 \label{sect:systemoverview}
 
- The system requires an operating ComCam at the Summit or on a test stand
-(either at the Base or Tucson). ~LDM-503-06 will have already shown that
-data acquisition (DAQ), archiver, header service, transfer mechanism,
-and ingest to the DBB are functional. ~Those systems are re-verified by
-this test as: 1) changes will almost certainly have occurred and 2) for
-this test to succeed the data must be properly formed and ingested for
-processing to succeed.\\[2\baselineskip]
 
 \subsection{Applicable Documents}\label{applicable-documents}
 
 \citeds{LDM-503}: Data Management Test Plan\\
-\citeds{LDM-639}: Data Management Acceptance Test Specification
+\citeds{LDM-639}: Data Management Acceptance Test Specification \\
+\citeds{LSE-61}: Data Management Requirements Specification \\
 
 
 \subsection{Document Overview}
 \label{sect:docoverview}
 
 This document was generated from Jira, obtaining the relevant information from the
-\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testPlan/LVV-P76}{LVV-P76}
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testPlan/LVV-P76}{LVV-P72}
```

Comment:
This seems to link to LVV-P76?

That must be why the last sentence of Sec. 1.4 says "The current status of test plan LVV-P76 in Jira is Completed.", and the Section 2 material is all from LVV-P76.

Reply:
argh  - wrong cycle too